types/host.go

```diff
@@ -25,6 +25,7 @@ import "time"
 type Host interface {
 	CPUTimer
 	Info() HostInfo
+	LoadAverage() (*LoadAverageInfo, error)
```

Comment:
I think this should not be added to the `Host` interface. It's an optional thing given that platforms like Windows do not provide it. Callers that want to use it would check if it's available for their platform like

```
	if v, ok := host.(types.LoadAverage); ok {
		load, err := v.LoadAverage()
		if err != nil {
			return err
		}
		fmt.Printf("Load: %#v\n", load)
	}
```

Once you do this then the aix/windows implementations that return errors should go away.

Reply:
Done! To be honest it makes more sense that adding empty implementations on each host, thanks for the heads up.

Please, have a deep look at the `system_test.go` file. I don't really know if it's correct to add the invocation of `LoadAverage` to the `TestHost` function. If not, let me know the best place to add the invocation.